sw/airborne/modules/actuators/actuators.h

```diff
@@ -57,10 +57,17 @@ struct act_feedback_t {
 extern uint32_t actuators_delay_time;
 extern bool   actuators_delay_done;
 
+// Actuator feedback structure for ABI Message
+struct actuator_t {
+  pprz_t pprz_val;                  ///< Actuator value in PPRZ units
+  int16_t driver_val;               ///< Actuator value in driver units (scaling from servo in airframe.h)
+};
```

Comment:
This is nice to have an actuator structure, and it could / should hold even more information, like the type of actuators, the status, its position or speed if measured, etc. Should we start already with this (and maybe change it later again), or already try to add future elements, even if not used yet ?

Reply:
It is very easy to add things later. I'd rather only add variables while that are used.
In this PR, as a first step, my goal is to be able to run the simulator directly from actuators everywhere (both INDI and rotorcraft)

Reply:
But I agree the rpm/position feedback could move to this struct a.s.a.p.  
Anybody able to help?

Reply:
I was planing to do this after finishing the reorganization of stabilization, if I can finish that someday...

Reply:
stabilization reorganization merged.